### ISSUE TITLE
Doesn't work on (at least) AWS images for Windows Server 2016

### ISSUE THREAD

**Issue by tomfanning** [DarwinJS/ChocoPackages#47]
On Windows Server 2016, executing the one-liner in the INSTALL file ([scenario 4](https://github.com/DarwinJS/ChocoPackages/blob/master/openssh/readme.md#install-scenario-4-complete-offline-install-wout-chocolatey-wout-wow64-wout-powershell-5-wout-internet)) fails as follows.

This is on AWS, completely fresh instance of ami-06bcae62.

```
****************************************************
Quick Config by Darwin Sanoy...
Win32-OpenSSH install on EC2AMAZ-MPJBS61
Changes to be made:
  [1] Sets PowerShell Execution Policy to "RemoteSigned"
  [2] Installs chocolatey package manager
  [3] Installs the chocolatey package openssh
****************************************************
Getting Started...

Name                           Version          Source           Summary
----                           -------          ------           -------
nuget                          2.8.5.208        https://onege... NuGet provider for the OneGet meta-package manager

Name         : chocolatey
Location     : http://chocolatey.org/api/v2/
Source       : chocolatey
ProviderName : NuGet
Provider     : Microsoft.PackageManagement.Implementation.PackageProvider
IsTrusted    : False
IsRegistered : True
IsValidated  : True
Details      : {}

openssh                        0.0.21.0         chocolatey       Open SSH tools for Windows - works in all CLIs.
dir : Cannot find path 'C:\Program Files\nuget\packages' because it does not exist.
At line:53 char:55
+ ... enssh." + "$((dir "$env:ProgramFiles\nuget\packages\openssh*" | %{[ve ...
+                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Program Files\nuget\packages:String) [Get-ChildItem], ItemNotFoundEx
   ception
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetChildItemCommand

cd : Cannot find path 'C:\Program Files\nuget\packages\openssh.\tools' because it does not exist.
At line:53 char:1
+ cd ("$env:ProgramFiles\nuget\packages\openssh." + "$((dir "$env:Progr ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Program File...\openssh.\tools:String) [Set-Location], ItemNotFoundE
   xception
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.SetLocationCommand

. : The term '.\barebonesinstaller.ps1' is not recognized as the name of a cmdlet, function, script file, or operable
program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:55 char:2
+ .".\barebonesinstaller.ps1" -SSHServerFeature
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (.\barebonesinstaller.ps1:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

**Comment by DarwinJS** [comment]
Ok, I usually test these instructions on Nano and on Server 2016 Nuget has an extra folder level.

I have fixed this in the latest version of https://raw.githubusercontent.com/DarwinJS/ChocoPackages/master/openssh/InstallChoco_and_win32-openssh_with_serverPS5.ps1

and to the Dockerfile here:
https://github.com/DarwinJS/ChocoPackages/blob/master/openssh/Dockerfile

and the manual instructions on https://github.com/DarwinJS/ChocoPackages/blob/master/openssh/readme.md

**Comment by DarwinJS** [comment]
Tested as fixed in 0.0.22.0.  FYI - not amazon specific - was happening on any Server 2016.
